qemu/tests/cfg/block_iscsi_4kdisk.cfg

```diff
@@ -0,0 +1,50 @@
+- block_iscsi_4kdisk:
+    virt_test_type = qemu
+    type = block_iscsi_4kdisk
+    only Windows
+    start_vm = no
+    kill_vm = yes
+    kill_vm_gracefully = yes
+    kill_vm_on_error = yes
+    shutdown_cleanly = no
+    reboot_method = shell
+    images = "stg0"
+    guest_port_unattended_install = 12323
+    inactivity_watcher = error
+    inactivity_treshold = 1800
+    image_verify_bootable = no
+    image_copy_on_error = no
+    image_aio = native
+    unattended_delivery_method = cdrom
+    cdroms += " unattended"
+    serial_name = SYSTEM_DISK0
+    cd_format_fixed = ide
+    drive_index_unattended = 1
+    drive_index_cd1 = 2
+    boot_once = d
+    medium = cdrom
+    redirs += " unattended_install"
+    iothreads = iothread0
+    blk_extra_params_stg0 = "serial=${serial_name},iothread=${iothreads}"
+    force_create_image_stg0 = no
+    force_remove_image_stg0 = no
```

Comment:
force_remove_image seems to be used for lvm only, so it can be removed

Reply:
The os image is /var/tmp/4kdisk/os , it not exist after cleanup, So the image can not be removed.

Reply:
you have already set 'remove_image_stg0 = no', which can take care of it